bot/seasons/evergreen/trivia_quiz.py

```diff
@@ -0,0 +1,238 @@
+import asyncio
+import json
+import logging
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+from fuzzywuzzy import fuzz
+
+from bot.constants import Roles
+
+
+logger = logging.getLogger(__name__)
+
+
+ANNOYED_EXPRESSIONS = ["-_-", "-.-"]
+
+WRONG_ANS_RESPONSE = [
+    "No one gave the correct answer",
+    "Better luck next time"
+]
+
+
+class TriviaQuiz(commands.Cog):
+    """A cog for all quiz commands."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        self.questions = self.load_questions()
+        self.game_status = {}
+        self.game_owners = {}
+        self.question_limit = 3
+        self.categories = {
+            "general": "Test your general knwoledge"
+            # "retro": "Questions related to retro gaming."
+        }
+
+    @staticmethod
+    def load_questions() -> dict:
+        """Load the questions from  json file."""
+        p = Path("bot", "resources", "evergreen", "trivia_quiz.json")
+        with p.open() as json_data:
+            questions = json.load(json_data)
+            return questions
+
+    @commands.command(name="quiz")
+    async def quiz_game(self, ctx: commands.Context, option: str, category: str = "general") -> None:
```

Comment:
I would recommend removing the `option` argument. It's not necessary, as you can just ensure you check if there's a session already ongoing (as you do already in line 68).